config:
  searcher:
    proc: 0
    solver: "ransac_solver"
    scenes: ["surface"]
    scene_frame_ids: ["camera/capture0"]
    master_frame_ids: ["camera/master0"]
    solve_frame_id: "camera/capture0"
  cropper:
    base_frame_id: "base"
    capture_frame_id: ["camera","flange"]
    source_frame_id: "camera/capture"
    frame_id: "camera/capture0"
    relay: "/sensors/X1"
    ansback: "/sensors/Y1"
  picker:
    multiplex: 1
    solve_frame_id: "camera/capture0"
    reference_frame_id: "camera/capture0"
    base_frame_id: "base"
  rsocket:
    base_frame_id: "flange"

